index.html

```diff
@@ -192,7 +192,7 @@ <h2>About Myself</h2>
                 <div class="code-experience">
                     <div class="list-header">
                         <ul class="experience-nav">
-                            <li>Languajes</li>
+                            <li>Languages</li>
                             <li><i class="fa-solid fa-angle-down"></i></i></li>
                         </ul>
```

Comment:
- [ ] Nice implementation of previous review. please stick to the Figma and control overlaping while changing from desktop version to the mobile version give it a margin and padding left.
## your design👇
![overlap](https://user-images.githubusercontent.com/66992901/193147117-08450d3c-c9bf-425b-aa16-8da925c76c98.PNG)

Reply:
Hi @sajid-munawar it's done!